### PR TITLE
Make minimized messaging conversations move above scroll-to-bottom

### PR DESCRIPTION
**Pull request by nightpool**
before:
image
after:
image

### CODE REVIEW

File: Extensions/scroll_to_bottom.css [hunk 1]
```diff
@@ -36,4 +36,8 @@
   height: 5px;
   bottom: 0;
   background: #EDC707;
+}
+
+.messaging-minimized-conversations {
+  margin-bottom: 60px;
 }
```
Comment (finagle29) [inline Extensions/scroll_to_bottom.css]: nitpick: no trailing newline?

Comment (nightpool) [inline Extensions/scroll_to_bottom.css]: it wasn't in the original file. will change if you think I should

Comment (finagle29) [inline Extensions/scroll_to_bottom.css]: it's in our .editorconfig so i think that you should change it

**Comment by nightpool** [comment]
done

**Comment by finagle29** [comment]
works, LGTM @homu r+

**Pull request merged**